modules/efk-logging-fluentd-external.adoc

```diff
@@ -77,29 +87,19 @@ not a path on your host system.
 ====
 +
 When configuring the external aggregator, it must be able to accept messages
-securely from Fluentd.
-
-. If the external aggregator is another Fluentd server, it must have the
-`fluent-plugin-secure-forward` plug-in installed and make use of the input
-plug-in it provides:
+securely from Fluentd and must have the `fluent-plugin-secure-forward` plug-in 
+installed and make use of the input plug-in it provides:
 +
 ----
 <source>
-  @type secure_forward
-
-  self_hostname ${HOSTNAME}
-  bind 0.0.0.0
-  port 24284
-
-  shared_key thisisasharedkey
-
-  secure yes
-  cert_path        /path/for/certificate/cert.pem
-  private_key_path /path/for/certificate/key.pem
-  private_key_passphrase secret_foo_bar_baz
+@type forward
```

Comment:
this is the old, insecure `forward` plugin config - use this instead: https://github.com/openshift/origin-aggregated-logging/blob/master/fluentd/configs.d/user/secure-forward.conf

Reply:
Here is an example of an `in_forward` configuration for Fluentd 1.0 or later:
```
<source>
  @type forward
  @id my-secure-forward
  port 24284
  bind 0.0.0.0
  <security>
    self_hostname server.fqdn.example.com
    shared_key    <shared_key_between_forwarder_and_forwardee>
  </security>
  <transport tls>
    cert_path        /path/to/my/tlsservercert
    private_key_path /path/to/my/tlsserverkey
    private_key_passphrase not_used_if_key_is_unencrypted
  </transport>
</source>
```